BasicPlaybackSampleApp/app/src/main/java/com/ooyala/sample/players/BasicPlaybackVideoPlayerActivity.java

```diff
@@ -95,6 +102,48 @@ public void update(Observable arg0, Object arg1) {
       return;
     }
 
+  // Hook to write Notifications to a temporary file on the device/emulator
+    // Keeps track of incoming notifications and makes sure count is right
+    count++ ;
```

Comment:
Sure , i can do that .

Let me know if below structure is fine .

Under com.ooyala.sample.utils , will add another file lets say ,ex,
logEventsToSdcard

Will put following lines as a function under that instead of in the
'update' function of BasicPlaybackVideoPlayerActivity.java file and will
call that function .

// Hook to write Notifications to a temporary file on the device/emulator
  // Keeps track of incoming notifications and makes sure count is right
  count++ ;
  String text="Notification Received: " + arg1 + " - state: " +
player.getState() + "count: " +count;

  //Empty Logcat buffer
  if(count==1) {

    try {
      Process process = new ProcessBuilder().command("logcat",
"-c").redirectErrorStream(true).start();
    } catch (IOException e) {
    }
  }

  //Writing events into file on device
  File logFile = new File("sdcard/log.file");
  if (!logFile.exists())
  {
    try
    {
      logFile.createNewFile();
    }
    catch (IOException e)
    {
      // TODO Auto-generated catch block
      e.printStackTrace();
    }
  }
  try
  {
    //BufferedWriter for performance, true to set append to file flag
    BufferedWriter buf = new BufferedWriter(new FileWriter(logFile, true));
    buf.append(text);
    buf.newLine();
    buf.close();
  }
  catch (IOException e)
  {
    // TODO Auto-generated catch block
    e.printStackTrace();
  }




On Fri, Jan 8, 2016 at 10:58 AM, lental <notifications@github.com> wrote:

> In
> BasicPlaybackSampleApp/app/src/main/java/com/ooyala/sample/players/BasicPlaybackVideoPlayerActivity.java
> <https://github.com/ooyala/android-sample-apps/pull/43#discussion_r49223757>
> :
>
> > @@ -95,6 +102,48 @@ public void update(Observable arg0, Object arg1) {
> >        return;
> >      }
> >
> > +  // Hook to write Notifications to a temporary file on the device/emulator
> > +    // Keeps track of incoming notifications and makes sure count is right
> > +    count++ ;
>
> any chance you can abstract this into a separate Utils/ file?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ooyala/android-sample-apps/pull/43/files#r49223757>.
>



-- 
Regards,

BinduMadhav Sondur
310, Crescent Village Circle , #2110 ,
San Jose , CA - 95134

PH: 315-247-5338